web/styles/message_row.css

```diff
@@ -384,7 +371,7 @@ $message_box_margin: 3px;
                 /* Don't adjust the line-height for collapsed or
                    edited/source messages when there's a sender,
                    as the sender text is always visible. */
```

Comment:
Is this comment still accurate?

Reply:
Oh, good catch. I'd meant to address that and it slipped my mind. We might be able to get rid of this comment and redundant selector entirely. Let me check and update the PR here accordingly.

Reply:
Confirmed: that whole extra selector can be done away with, so I've done just that. This makes for a nice additional bit of cleanup on the `.message-time` selector that was begun in #29501.